src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

no need to repeat 1 column
```suggestion
  grid-template-columns: 200px;
```